qemu/tests/qemu_guest_agent.py

```diff
@@ -4029,6 +4049,16 @@ def gagent_check_get_disks(self, test, params, env):
                 test.fail("Disk %s name is different "
                           "between guest and qga." % diskname)
 
+            error_context.context("Check disk bus type", LOG_JOB.info)
+            bt_guest = session.cmd_output(params['cmd_bustype_guest']
+                                          % diskname).strip().split()[0]
+            bt_qga = disk_info['address']['bus-type']
+            if bt_qga.upper() != bt_guest:
+                if bt_qga != "sas" and bt_guest != "SCSI":
```

Comment:
@6-dehan Could you please elaborate on this exception?

Reply:
I have to run it and then get the result. will comment at that time.

Reply:
this is indeed a nice shot. it means I have to change a new command to query 'bustype' inside guest.  the original command printout incorrect value now. let me try the new command. I will update after that.